app/views/assignments/_boot.js.erb

```diff
@@ -52,7 +52,7 @@
           <label class="required" for="'${input_id}'">${I18n.t('attributes.filename')}</label>
           <input type="text"
                  required="required"
-                 pattern="^[\-\._a-zA-Z0-9][\-\/\._a-zA-Z0-9\\s]*$"
+                 pattern="^[\\-\\._a-zA-Z0-9][\\-\\/\\._a-zA-Z0-9\\\\s]*$"
```

Comment:
This change isn't the one I was referring to. (In a regex, `-` at the beginning of a character class doesn't need to be escaped, nor does `.`)

Try adding a required file and then clicking the "delete" link to delete it, that's what caused me the error.